src/styles/includes/typography.scss

```
@@ -82,7 +82,7 @@
   }
 
   a {
-    color: var(--color-green-dark);
+    color: color('color-green-dark');
```

After changing the `color()` function, this would just be:
```suggestion
    color: color('green-dark');
```